packages/communication-react/review/communication-react.api.md

```diff
@@ -1199,6 +1216,18 @@ export const newClearErrorsModifier: (targets: ChatErrorTargets[]) => ChatStateM
 // @public (undocumented)
 export type OmitNever<T> = Pick<T, AllKeys<T>[keyof AllKeys<T>]>;
 
+// @public
+export type OnRenderAvatarType = (userId?: string, options?: {
+    hidePersonaDetails?: boolean;
+    initialsTextColor?: string;
+    presence?: PersonaPresence;
+    size?: PersonaSize;
+    coinSize?: number;
+    text?: string;
+    styles?: IStyleFunctionOrObject<IPersonaStyleProps, IPersonaStyles>;
+},
+defaultOnRender?: (props: PlaceholderProps) => JSX.Element) => JSX.Element;
```

Comment:
I'm probably being naive here: Would it be possible for the `userId` and `options` here to simply be replaced with `PlaceholderProps`? Those are all the arguments that our internal implementation to render the avatar needs, so we give the exact same arguments to Contoso. In the simplest case, they can simply pass those through to `defaultOnRender`. In more complex cases, every business logic that we can implement in our default implementation potentially requires an override in Contoso's implementation, and they get the exact same arguments to act on.

This is a more general question: For each render callback where we have a default implementation, should the arugment to the render callback be identical to the arguments for the default implementation?

Reply:
PlaceholderProps is only being used in VideoTile currently. 
I added it as a 3rd argument to not have to re-write a whole bunch of code.
userId, and options play nicely when you are not trying to customize a VideoTile component.

Reply:
And no to your more general question